models/Document/PageSnippet.php

```diff
@@ -113,7 +113,7 @@ protected function update($params = [])
     /**
      * @param bool $setModificationDate
      * @param bool $saveOnlyVersion
-     * @param $versionNote string version note
+     * @param string $versionNote
```

Comment:
"version note" description missing

Reply:
I would say the variable name speaks for itself, so it's ok.

Reply:
Yes I also thought that the description has no added value